demo/grpc-checks/example/example.go

```diff
@@ -0,0 +1,32 @@
+package example
+
+import (
+	"context"
+	"log"
+
+	// maybe just use the google module?
+	ghc "github.com/hashicorp/nomad/demo/grpc-checks/health"
+)
+
+// Server is a trivial gRPC server that implements the standard grpc.health.v1
+// interface.
+type Server struct {
+}
+
+func New() *Server {
+	return new(Server)
+}
+
+func (s *Server) Check(ctx context.Context, hcr *ghc.HealthCheckRequest) (*ghc.HealthCheckResponse, error) {
+	log.Printf("Check:%s (%s)", hcr.Service, hcr.String())
+	return &ghc.HealthCheckResponse{
+		Status: ghc.HealthCheckResponse_SERVING,
```

Comment:
Any ideas for a simple way we could toggle the status remotely. I think that would be helpful in a demo. Something as simple as checking if a file exists and we could `nomad exec ... touch blah` to start making the health check fail.

Reply:
Yeah I was thinking about this already, maybe have it intercept SIGINT and toggle the pass/fail mode?